mod_test.ts

```diff
@@ -949,25 +949,47 @@ const TXT_PLUGIN: esbuildNative.Plugin = {
   },
 };
 
-Deno.test("txt plugin", async (t) => {
+Deno.test("css plugin", async (t) => {
   +await testLoader(t, LOADERS, async (esbuild, loader) => {
     const res = await esbuild.build({
       ...DEFAULT_OPTS,
       plugins: [
-        denoResolverPlugin(),
-        TXT_PLUGIN,
-        denoLoaderPlugin({ loader }),
+        ...denoPlugins({ loader }),
+        CSS_PLUGIN,
```

Comment:
Now the plugin no longer needs to be between the denoResolverPlugin and denoLoaderPlugin. You could actually put CSS_PLUGIN before or after all denoPlugins. I just chose to put it after, but both work.